.travis.yml

```diff
@@ -160,5 +151,7 @@ script:
     fi
   - if [[ $TEST_TARGET == 'coding' ]]; then
       cd $INSTALL_DIR;
+      python setup.py pyke_rules;
+      python setup.py std_names;
```

Comment:
At this point, (I think) it's too late to compile the rules, as it'll put the compiled rules in the `lib` (if anywhere) and won't be installed (which has already happened)

Reply:
Aha, but this is in the ``coding`` block, which actually checks the ``source``, not the installed version. Caught me out too!

Reply:
But it's the travis `coding` pep8 tests that are failing ... hinting (to me) that the compiled files aren't where they should be ... ? They might be in the `lib` source, but aren't the tests running against the installed version of iris?

Reply:
They weren't for standard_names (at least, that is what I observed in an earlier commit!)

Reply:
Hmmm something isn't quite as it seems. Not quite sure what's going on ...